File: exercises/small_problems_101/easy_3/palindromic_strings_1.rb
def palindrome?(input)
  input.reverse == input
end

# def palindrome?(sentence)
#   input = sentence.split(%r{\s*})  
#   reverse_input = []
#   loop do
#     reverse_input.push(input.pop)
#     break if input.size == 0
#   end
#   reverse_input == sentence.split(%r{\s*})  
# end


puts palindrome?('madam') == true
puts palindrome?('Madam') == false          # (case matters)
puts palindrome?("madam i'm adam") == false # (all characters matter)
puts palindrome?('356653') == true

File: frontend/javascript/exercises/easy_3/palindromic_strings_1.js
function isPalindrome(str) {
  return str.split('').reverse().join('') === str;
}

console.log(isPalindrome('madam') === true);                    // true
console.log(isPalindrome('Madam') === false);                    // false (case matters)
console.log(isPalindrome('madam i\'m adam') === false);          // false (all characters matter)
console.log(isPalindrome('356653') === true);                   // true